packages/buefy-next/src/components/upload/Upload.vue

```diff
@@ -70,7 +73,12 @@ export default {
             default: false
         }
     },
-    emits: ['invalid', 'update:modelValue'],
+    emits: {
+        /* eslint-disable @typescript-eslint/no-unused-vars */
+        invalid: (valid: boolean) => true,
```

Comment:
In reply to [this comment](https://github.com/ntohq/buefy-next/pull/269#discussion_r1732968080), you should remove the `valid` parameter here.